plugins/module_utils/docker/swarm.py

```diff
@@ -271,9 +271,9 @@ def get_service_inspect(self, service_id, skip_missing=False):
         except APIError as exc:
             if exc.status_code == 503:
                 self.fail("Cannot inspect service: To inspect service execute module on Swarm Manager")
-            self.fail("Error inspecting swarm service: %s" % exc)
+            self.fail("Error inspecting swarm ansible.builtin.service: %s" % exc)
```

Comment:
There are two more changes in this file which need to be reverted.

Reply:
good eye, thanks:)